src/install/kafka/default-install-linux.mdx

```diff
@@ -0,0 +1,48 @@
+---
+componentType: default
+headingText: Configure the Kafka integration
+---
+
+1. Change directory to the integrations configuration folder by running:
```

Comment:
Probably should be "Change your directory", reads kind of wonky without the "your" here.